scripts/get-good-commit.py

```diff
@@ -62,12 +62,16 @@ def get_good_commit(
         logging.info(f"Found good commit: {commit_url}")
         return commit.sha
 
-    return ""
+    sha = repo.get_commit(sha=start_ref).sha
+    logging.info(
+        f"Not found a good commit, using the intial one: {start_ref}, aka {sha}"
+    )
+    return {sha}
```

Comment:
```suggestion
    return sha
```
Or is this some weird new python syntax I don't know about?

Reply:
No you're right. I don't know why I've used that. Probably because I starred at the f-strings.